comments.php

```diff
@@ -38,7 +38,7 @@
 
 		?>
 
- 	</section>
+	 </section>
```

Comment:
Looks like there is an extra space after the tab on this line.

Reply:
(As a side note, CodeSniffer suggested that all `<?php` and `?>` should be placed on their own lines but that seemed to radical to me, so I discarded all those suggestions.)

Reply:
I'm kind of torn on this one. On one hand I don't really like how it looks starting a line with a closing php tag (`?></section>`) and having stuff come after it on the same line. On the other hand it uses up more space to have the php tags on their own lines. 

Either way, the opening and closing tags should be consistent so one of them should be moved. The codesniffer didn't say anything about the opening or closing tag here?

Reply:
> The codesniffer didn't say anything about the opening or closing tag here?

Yes, it did. CodeSniffer suggested that all <?php and ?> should be placed on their own lines but that seemed too radical to me, so I discarded all those suggestions for now. But looking at the [section for opening and closing php tags in the WordPress php code standard](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#opening-and-closing-php-tags) it is actually very clear:

> When embedding multi-line PHP snippets within a HTML block, the PHP open and close tags must be on a line by themselves.

It's just that we have a lot code like `<?php while ( have_posts() ) : the_post(); ?>` that CodeSniffer was breaking up in multiple lines and I wasn't sure we were ready for that.

Reply:
@Aetles I think the rule should be if the opening and closing php tags appear on the same line then it's ok, otherwise put them on their own line.

I looked over the PR again and I think if you just remove the space on line 41 and put the opening php tag on line 15 on its own line, we are good to merge.